web-ng/src/app/components/drawing-tools/drawing-tools.component.ts

```diff
@@ -77,13 +81,16 @@ export class DrawingToolsComponent implements OnInit, OnDestroy {
     this.lang = 'en';
     this.layers$ = projectService.getActiveProject$().pipe(
       tap(project => {
+        this.activeProject = project;
         this.selectedLayerId = project.layers.keySeq().first();
         this.drawingToolsService.setSelectedLayerId(this.selectedLayerId);
       }),
       map(project =>
         List(project.layers.valueSeq().toArray())
           .sortBy(l => l.index)
-          .filter(l => projectService.canUserAddPointsToLayer(l))
+          .filter(l =>
+            authService.canUserAddPointsToLayer(this.activeProject, l)
+          )
```

Comment:
This doesn't seem to have anything to do with the project list; was this change accidentally included in this PR? If so please move into its own PR.

Reply:
https://github.com/google/ground-platform/pull/643#discussion_r667066544